services/movie.py

```diff
@@ -22,6 +27,7 @@ def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
```

Comment:
The `get_movie_by_id` function does not handle the case where a movie with the given ID does not exist. Consider using `get_object_or_404` or handling the `Movie.DoesNotExist` exception to provide a more user-friendly error message.